src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -102,6 +102,15 @@ namespace System
         // this constant is -79,228,162,514,264,337,593,543,950,335.
         public const decimal MinValue = -79228162514264337593543950335m;
 
+        /// <inheritdoc cref="IAdditiveIdentity{TSelf, TResult}.AdditiveIdentity" />
+        public const decimal AdditiveIdentity = 0m;
+
+        /// <inheritdoc cref="IMultiplicativeIdentity{TSelf, TResult}.MultiplicativeIdentity" />
+        public const decimal MultiplicativeIdentity = 1m;
+
+        /// <inheritdoc cref="ISignedNumber{TSelf}.NegativeOne" />
+        public const decimal NegativeOne = -1m;
```

Comment:
Decimal already defines and exposes `MinusOne`. This will need to be discussed in API review.

My own preference is `MinusOne` is the "incorrect" name and we shouldn't propagate it to the other types or interfaces.